publication-commons/src/main/java/no/unit/nva/publication/service/impl/ResourceServiceUtils.java

```diff
@@ -29,21 +29,25 @@
         PARTITION_KEY_NAME_PLACEHOLDER + " = " + PARTITION_KEY_VALUE_PLACEHOLDER
         + " AND "
         + SORT_KEY_NAME_PLACEHOLDER + " = " + SORT_KEY_VALUE_PLACEHOLDER;
-
+    
     public static final Map<String, String> PRIMARY_KEY_EQUALITY_CONDITION_ATTRIBUTE_NAMES =
         primaryKeyEqualityConditionAttributeNames();
-
+    
     public static final String KEY_NOT_EXISTS_CONDITION = keyNotExistsCondition();
     public static final String UNSUPPORTED_KEY_TYPE_EXCEPTION = "Currently only String values are supported";
-
+    
     private ResourceServiceUtils() {
     }
-
+    
+    //Use methods found in Dao
+    @Deprecated
```

Comment:
In this PR, these methods are copied in Dao.java. In subsequent PR, they are deleted